projects/github.com/edenhill/kcat/package.yml

```diff
@@ -0,0 +1,29 @@
+distributable:
+  url: https://github.com/edenhill/kcat/archive/refs/tags/{{version}}.tar.gz
+  strip-components: 1
+versions:
+  github: edenhill/kcat
+dependencies:
+  apache.org/avro: '*'
+  github.com/confluentinc/librdkafka: '*'
+  github.com/confluentinc/libserdes: '*'
+  lloyd.github.io/yajl: '*'
+build:
+  dependencies:
+    linux:
+      gnu.org/gcc: '*'
+      gnu.org/make: '*'
```

Comment:
gnu/make is always available during builds now

Reply:
even if adding a compiler? good change!